### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by coldfire7** [pavelgar/pihello#5]
Tried it on 2 different systems and got the same error
```
# pihello 127.0.0.1
Traceback (most recent call last):
  File "/usr/local/bin/pihello", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.9/site-packages/pihello/cli.py", line 112, in main
    summary = flatten_dict(get_data(pihole))
  File "/usr/local/lib/python3.9/site-packages/pihello/cli.py", line 73, in flatten_dict
    for k, v in d.items():
AttributeError: 'list' object has no attribute 'items'
```

**Comment by WillPresley** [comment]
This is fixed by my PR #4, but it doesn't seem that @pavelgar has checked it out yet. If you are comfortable pulling from that commit and building yourself, it will go back to working as it should!

**Comment by coldfire7** [comment]
> This is fixed by my PR #4, but it doesn't seem that @pavelgar has checked it out yet. If you are comfortable pulling from that commit and building yourself, it will go back to working as it should!

Thanks! made the necessary changes to the `cli.py` file, and it's working.

![2023-01-29_22-31-27](https://user-images.githubusercontent.com/533476/215340680-cfc0d198-0c5b-44ee-a951-92b31c05de51.png)

**Comment by pavelgar** [comment]
Hi! Apologies for not checking-in more regularly.

PR #4 looks good to me! Thanks @WillPresley
